node_modules/titanium-sdk/lib/emulators/genymotion.js

```diff
@@ -286,7 +286,10 @@ function findGenymotion(dir, config, callback) {
 	if (!player || !fs.existsSync(player)) {
 		player = path.join(dir, 'player' + exe);
 	}
-	if (player && !fs.existsSync(player)) {
+	if (!fs.existsSync(player) && process.platform != 'win32') {
```

Comment:
Well, it works on Linux as it was since the `fs.existsSync` on the next line will simply return false, but here's the, uh...  "forward port" for master:

https://github.com/appcelerator/titanium_mobile/pull/7670